src/output.cpp

```diff
@@ -1958,8 +1958,9 @@ void play_music(std::string)
 {
 }
 
-void play_sound(std::string)
+void play_sound_effect(std::string id, std::string variant, int volume)
 {
+    (void)id;(void)variant;(void)volume;
```

Comment:
However, the parameters are actually used in the alternative branch. I could do this:

```
#ifdef SDL_SOUND
void play_sound_effect(std::string id, std::string variant, int volume) {
    sound_effect_entry *effect_to_play = get_sound_effect(id, variant);
    if (!effect_to_play) {
        return;
    }
    Mix_VolumeChunk(effect_to_play->chunk, effect_to_play->volume * OPTIONS["SOUND_EFFECT_VOLUME"] * volume / (100 * 100));
    if (Mix_PlayChannel(-1, effect_to_play->chunk, 0) == -1) {
        dbg( D_ERROR ) << "Failed to play sound effect: " << Mix_GetError();
    }
#else
void play_sound_effect(std::string id, std::string, int) {
}
#endif
```

But that's very redundant. Got an idea for a better way?